benchmark/smallbank/config.json

```diff
@@ -0,0 +1,38 @@
+{
+   "blockchain": {
+        "type": "fabric",
+        "config": "benchmark/smallbank/fabric.json"
+  },
+  "command" : {
+    "start": "docker-compose -f network/fabric/simplenetwork/docker-compose.yaml up -d",
+    "end" : "docker-compose -f network/fabric/simplenetwork/docker-compose.yaml down;docker rm $(docker ps -aq)"
+  },
+  "test": {
+    "name": "smallbank",
+    "description" : "This is smallbank benchmark for caliper",
+    "clients": {
+      "type": "local",
+      "number": 5
+    },
+    "rounds": [{
+        "label" : "smallOperations",
+        "txNumber" : [2000],
+        "rateControl" : [{"type": "fixed-rate", "opts": {"tps" : 200}}],
+        "arguments": {"accounts":300, "txnPerBatch":10},
```

Comment:
If we set txnPerBatch = 1 for fabric, will it fix 'Invalid transaction: MVCC_READ_CONFLICT' error in fabric?

Reply:
Unfortunately no...... But at least there will be some time interval between two adjacent submittings, so it may be better at low submitting rate.